launch_testing_ros/launch_testing_ros/tools/output.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018 Open Source Robotics Foundation, Inc.
```

Comment:
Is it not 2019?
Or this came from another file?

Reply:
It did came from somewhere else but 2019 is still correct.